internal/cli/apis.go

```diff
@@ -234,7 +244,7 @@ func createAPICmd(cli *cli) *cobra.Command {
 				return err
 			}
 
-			defaultTokenLifetime := strconv.Itoa(apiDefaultTokenLifetime())
+			defaultTokenLifetime := strconv.Itoa(apiDefaultTokenLifetime)
```

Comment:
Looks like a mistake:
```suggestion
			defaultTokenLifetime := strconv.Itoa(apiDefaultTokenLifetime())
```